classes/FeatureFlag.php

```diff
@@ -77,6 +82,12 @@ class FeatureFlagCore extends ObjectModel
         'primary' => 'id_feature_flag',
         'fields' => [
             'name' => ['type' => self::TYPE_STRING, 'required' => true, 'size' => 191],
+            'type' => [
+                'type' => self::TYPE_STRING,
+                'required' => true,
+                'size' => 64,
+                'default' => FeatureFlagSettings::TYPE_DEFAULT,
+            ],
```

Comment:
Do we have any validation about the allowed values?

Reply:
Not in the ObjectModel but I'm not sure if it's needed This object model is mostly used so that we can load the XML fixtures on install (which is based on ObjectModel not Doctrine) It's the only place where this class is used and we know what fixtures are contained so I don't think we would introduce some false values

Reply:
It's not blocking but I don't think the object model should consider external uses. It doesn't know about who is using it.